Lib/difflib.py

```diff
@@ -1632,13 +1632,22 @@ def _line_pair_iterator():
 </html>"""
 
 _styles = """
+        :root {color-scheme: light dark}
         table.diff {font-family:Courier; border:medium;}
         .diff_header {background-color:#e0e0e0}
         td.diff_header {text-align:right}
         .diff_next {background-color:#c0c0c0}
-        .diff_add {background-color:#aaffaa}
+        .diff_add {background-color:palegreen}
         .diff_chg {background-color:#ffff77}
-        .diff_sub {background-color:#ffaaaa}"""
+        .diff_sub {background-color:#ffaaaa}
+
+        @media (prefers-color-scheme: dark) {
+            .diff_header {background-color:#666}
+            .diff_next {background-color:#444}
+            .diff_add {background-color:darkgreen}
+            .diff_chg {background-color:goldenrod}
```

Comment:
Thanks for this, looks good!

One suggestion: the contrast ratio of white on goldenrod is 2.23, and we should aim for at least 4.5 to meet WCAG AA accessibility guidelines:

<img width="390" alt="image" src="https://github.com/user-attachments/assets/6af20baa-033f-4d56-9441-a142b157a4ca" />

Please could you suggest another colour?

See also https://webaim.org/resources/contrastchecker/

And this one could be improved as well:

<img width="552" alt="image" src="https://github.com/user-attachments/assets/bbc7b61e-8b28-4f97-bfa1-0b9bf87e660a" />